modules/nw-modifying-operator-install-config.adoc

```diff
@@ -24,57 +24,40 @@ endif::[]
 [id="modifying-nwoperator-config-startup_{context}"]
 = Specifying advanced network configuration
 
-You can use advanced configuration customization to integrate your cluster into your existing network environment by specifying additional configuration for your cluster network provider.
+You can use advanced network configuration for your cluster network provider to integrate your cluster into your existing network environment.
 You can specify advanced network configuration only before you install the cluster.
 
 [IMPORTANT]
 ====
-Modifying the {product-title} manifest files created by the installation program is not supported. Applying a manifest file that you create, as in the following procedure, is supported.
+Customizing your network configuration by modifying the {product-title} manifest files created by the installation program is not supported. Applying a manifest file that you create, as in the following procedure, is supported.
 ====
 
 .Prerequisites
 
-* Create the `install-config.yaml` file and complete any modifications to it.
-ifdef::ignition-config[]
-* Create the Ignition config files for your cluster.
```

Comment:
@jboxman just to double check, is there a specific scenario where the Ignition configs must be created before this procedure? From reviewing the position of the procedure in the various assemblies and the steps themselves, my understanding is that the Ignition config files are typically created after this. For example, step 1 references the directory where the install-config.yaml file is located. If the Ignition config file has been generated, then that file would have been consumed. The same goes for the `<installation_directory>/manifest` directory and its contents. I thought it worth asking in case I have missed something relating to a conditional or something. Thanks!

Reply:
@pneedle-rh, the fun thing about installation docs is that it is a magic grab bag of random items.

I know almost nothing about installation, but at the time, some other customization installation assembly must have had this, so I copied it.

I don't know what it means, why it is there, or who might know about it, unfortunately.

I can certain go back to the original PR and see if there is any insight there as to why I might have done this, but it definitely originated elsewhere and at the time it seemed critical to include it here as well.

Reply:
@jboxman thank you for the background. For the bare metal scenario that particular prerequisite does not seem to fit for this procedure. If you would like me to add it back only for the vSphere and VMC conditionals, I can. Please let me know. Thanks!

Reply:
@pneedle-rh so this is where that originated from: 4c9929632521245eef90b9b597225566f66e1a21

> Add network customization for additional platforms

> - bare-metal
> - vSphere
> - GCP

So it looks like at that time, those procedures required the creation of Ignition configuration files prior to running this procedure.

But that's as much as I know; At the time, it looked like this additional step was required, so I included it. Because I don't generally work with installation content, I don't know the provenance of this. Maybe @kalexand-rh would know or can suggest someone that does?

So much has changed over the year, it's certainly possible that the prerequisite is now exposed somewhere that it should not be or that the reverse is true, but I don't have a line of sight into this.

Reply:
@jboxman thanks for the further details. I have just reviewed this module in all 8 of the assemblies that it is included in. From that review, I can't see any specific circumstances where the prerequisite to have created the Ignition config files seems relevant to the module.

In all cases, the first step is to change to the directory that contains the `install-config.yaml`, which would not exist if the Ignition config files had been created.

Additionally, in all cases, the first step is to create the manifests by using the `./openshift-install create manifests` command. That requires the `install-config.yaml` file and is done before the Ignition config file creation step.

Based on our discussion, I propose that we remove the prerequisite in the module for all conditional renderings.